pkg/datapath/loader/loader.go

```diff
@@ -42,6 +42,11 @@ const (
 	symbolFromEndpoint = "from-container"
 	symbolToEndpoint   = "to-container"
 
+	symbolFromHostEp1 = "from-netdev"
+	symbolToHostEp1   = "to-netdev"
+	symbolFromHostEp2 = "from-host"
+	symbolToHostEp2   = "to-host"
```

Comment:
I'm not sure what your point is here to be honest :worried: The behavior you're describing should be preserved by this pull request.

So far, I have only considered `cilium_host` and the native device as part of the host endpoint. But maybe it would make things more consistent/readable if I also move the `cilium_net` (i.e., `$HOST_DEV2`) attachment to the Go side.

> I'm only about 3 commits into the PR right now so maybe I'm missing future context as to why this makes sense. I guess I'll put down my assumptions and background and continue on :-)

Don't :-) Subsequent commits introduce the actual policy enforcement and are fairly independent. If things are unclear here, then I need to rework it.